migration/20180307-replace-recently-updated-index.sql

```diff
@@ -7,11 +7,5 @@ DO $$
             WHEN OTHERS THEN RAISE NOTICE 'mv_works_for_lanes_by_recently_updated did not previously exist.';
         END;
 	create index mv_works_for_lanes_by_recently_updated on mv_works_for_lanes (GREATEST(availability_time, first_appearance, last_update_time) DESC, collection_id, works_id);
-
-        BEGIN
-	    create index mv_works_for_lanes_list_and_collection_id on mv_works_for_lanes (list_id, collection_id);
-        EXCEPTION
-            WHEN duplicate_table THEN RAISE NOTICE 'Warning: mv_works_for_lanes_list_and_collection_id already exists.';
-        END;
```

Comment:
This index is created differently now, and there's no need to create it when it will just be deleted in a future migration script.